assets/js/googlesitekit/datastore/user/user-info.test.js

```diff
@@ -364,5 +364,53 @@ describe( 'core/user userInfo', () => {
 				expect( global[ userDataGlobal ] ).not.toEqual( undefined );
 			} );
 		} );
+
+		describe( 'getAccountChooserURL', () => {
+			it( 'throws an error if a destinationURL is not given', () => {
+				expect( () => {
+					registry.select( CORE_USER ).getAccountChooserURL();
+				} ).toThrow( 'destinationURL is required' );
+			} );
+
+			it( 'returns the encoded destination url with the email appended', async () => {
+				global[ userDataGlobal ] = userData;
+
+				const testURL = 'test destination url';
```

Comment:
Let's use a "more real" value that we'd expect here, like an actual URL instead of what looks like just a generic string.

```suggestion
				const testURL = 'https://analytics.google.com/dashboard/';
```

or something like that. I'm always a fan of using as-close-to-real data in tests so they're more readable and understandable at a glance. Plus: then we get to test the URI encoding here.